src/ray/ray_config_def.h

```diff
@@ -135,3 +135,6 @@ RAY_CONFIG(int, num_workers_per_process, 1);
 
 /// Maximum timeout in milliseconds within which a task lease must be renewed.
 RAY_CONFIG(int64_t, max_task_lease_timeout_ms, 60000);
+
+/// Maximum number of checkpoints to keep in GCS for an actor.
+RAY_CONFIG(uint32_t, num_actor_checkpoints_to_keep, 200);
```

Comment:
How about defaulting this to 2? If the application is writing its checkpoint data in place, then that would guarantee that the stored checkpoint is always in the backend's `available_checkpoints`. If the application isn't writing its checkpoint data in place and therefore needs to GC old checkpoints, then this would minimize the amount of checkpoint data per actor.